nbconvert/preprocessors/execute.py

```diff
@@ -146,4 +170,7 @@ def run_cell(self, cell):
             else:
                 outs.append(out)
 
+            if out.output_type == 'error' and self.abort_on_error:
```

Comment:
Should this be listening for 'error' output messages, or should it be checking the for an error status in `execute_reply` messages? The protocol doesn't force the two to be linked. I'd say that the execute_reply status is what really determines whether execution should proceed or not.